tests/free_surface_blob_nonzero_melt_simple.prm

```diff
@@ -0,0 +1,154 @@
+# Test for free surface topography due to a rising blob
+# with melt transport = on and nonzero porosity.
+# The test is the same as free_surface_blob_melt, except that melt
+# has a nonzero initial condition. 
+# We test if the boundary coditions for melt are set correctly
+# (no melt flowing in/out at the free surface) by checking if the
+# mass of the porosity field is conserved.
```

Comment:
This test has no free surface (top is free slip), so the description and name are a bit confusing to me. Also there is no test output.